core/src/main/java/org/wildfly/extras/creaper/core/online/KeyStoreType.java

```diff
@@ -0,0 +1,53 @@
+package org.wildfly.extras.creaper.core.online;
+
+import java.security.KeyStore;
+
+/**
+ * See http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#KeyStore
+ */
+public enum KeyStoreType {
+    /** The proprietary keystore implementation provided by the SunJCE provider. */
+    JCEKS("jceks"),
+    /** The proprietary keystore implementation provided by the SUN provider. */
+    JKS("jks"),
+    /** A domain keystore is a collection of keystores presented as a single logical keystore. */
+    DKS("dks"),
+    /** A keystore backed by a PKCS #11 token. */
+    PKCS11("pkcs11"),
+    /** The transfer syntax for personal identity information as defined in PKCS #12. */
+    PKCS12("pkcs12"),
+    /** The second version of PKCS12 type keystore. */
+    PKCS12S2("pkcs12s2"),
+    ;
```

Comment:
If some of the enum constants are impossible to use from our code (the PKCS11 thing looks particularly suspicious), they should either be removed or commented.